deployment/helm/resource-management-policies/topology-aware/values.yaml

```
@@ -18,3 +18,11 @@ hostPort: 8891
 resources:
   cpu: 500m
   memory: 512Mi
+
+nri:
+  patchContainerdConfig: false
+
+init:
```

nit: plain `init` looks somehow ambiguous to me. How about `initContainerImage` or `initImage` for example (we would then have `image` and `initContainerImage` fields )?

```suggestion
initContainerImage:
```

---

+1 for initContainerImage.

---

replaced to be `initContainerImage`